### PR TITLE
created nested npm install script

### PR DESCRIPTION
**Pull request by isose**
"npm install" now also runs a postinstall script that runs npm install in client and server folders

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,10 +25,5 @@
     "eslint-plugin-react-hooks": "^4.2.0",
     "npm-run-all": "^4.1.5"
   },
-  "dependencies": {
```
Comment (isose) [inline package.json:25]: Backend dependencies are already in the server/package.json, so removed them from here.

**Comment by isose** [comment]
[Create script for nested npm install](https://trello.com/c/Neik9dRy/26-create-script-for-nested-npm-install)

**Review by theninjawoohoo — APPROVED** [review]
I like the explanation, keep it up CEO.

**Review by AndyLiuCodes — APPROVED** [review]
LGTM

**Pull request merged**